lib/utils/extensions.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:pokedex/utils/strings.dart' as str;
+import 'package:pokedex/utils/constant.dart' as k;
+
+extension ModifyUrl on String {
+  String get toModifyUrl => k.imageURL.replaceAll(str.indexReplace, split('/')[6]);
```

Comment:
modifying url for what? that's why I suggested `toCustomUrl`, meaning => `to` (action that will happen for this function) `CustomUrl` (end product after the action is done) 🙂